src/stock-items/add-stock-item/transactions/printout/printable-transaction-header.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styles from './printable-transaction-header.scss';
+import { useConfig } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import startCase from 'lodash-es/startCase';
+
+interface PrintableTransactionHeaderProps {
+  itemName: string;
+}
+
+const PrintableTransactionHeader: React.FC<PrintableTransactionHeaderProps> = ({ itemName }) => {
+  const { t } = useTranslation();
+  const { logo } = useConfig({ externalModuleName: '@kenyaemr/esm-login-app' });
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.printableHeader}>
+        <p className={styles.heading}>{t('bincard', 'Bin Card')}</p>
+        {logo?.src ? (
+          <img className={styles.img} height={60} width={250} src={logo.src} alt={logo.alt} />
+        ) : logo?.name ? (
+          logo.name
+        ) : (
+          // OpenMRS Logo
```

Comment:
```tsx
   <svg role="img" className={styles.logo}>
      <title>{t('openmrsLogo', 'OpenMRS logo')}</title>
      <use href="#omrs-logo-full-color"></use>
    </svg> 
```

Reply:
We can use this instead duplicating the svg contents

Reply:
This method works when displaying the preview component but it does not print out when printed